rhp/v4/rhp.go

```diff
@@ -458,6 +465,26 @@ type (
 		Balance types.Currency `json:"balance"`
 	}
 
+	// RPCReplenishAccountsRequest is the request type for RPCReplenishAccounts.
+	RPCReplenishAccountsRequest struct {
+		Accounts           []Account            `json:"accounts"`
+		Target             types.Currency       `json:"target"`
+		ContractID         types.FileContractID `json:"contractID"`
+		ChallengeSignature types.Signature      `json:"challengeSignature"`
+	}
+	// RPCReplenishAccountsResponse is the response type for RPCReplenishAccounts.
+	RPCReplenishAccountsResponse struct {
+		Deposits []AccountDeposit `json:"deposits"`
+	}
+	// RPCReplenishAccountsSecondResponse is the second response type for RPCReplenishAccounts.
+	RPCReplenishAccountsSecondResponse struct {
+		RenterSignature types.Signature `json:"renterSignature"`
+	}
+	// RPCReplenishAccountsThirdResponse is the third response type for RPCReplenishAccounts.
+	RPCReplenishAccountsThirdResponse struct {
+		HostSignature types.Signature `json:"hostSignature"`
+	}
```

Comment:
could the host send their signature along with the deposits?

Reply:
It would be more efficient, but I believe we want to be consistent with the host sending their signature last since they are primarily responsible for maintaining and broadcasting the state.